### PR TITLE
Task/shank 48 sql52

### PR DESCRIPTION
**Pull request by jhulbert-va**
[SHANK-48](https://vasdvp.atlassian.net/browse/SHANK-48)

* 6 non-datamart resources revved to v1.02

### CODE REVIEW

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/location/LocationController.java [hunk 1]
```diff
@@ -76,12 +76,16 @@ public Location read(@PathVariable("publicId") String publicId) {
...
+   * The XML should remain the same, but the version of the resource needs to be incremented for
+   * SQL52.
+   */
   private CdwLocation100Root search(MultiValueMap<String, String> params) {
     Query<CdwLocation100Root> query =
         Query.forType(CdwLocation100Root.class)
             .profile(Profile.DSTU2)
             .resource("Location")
-            .version("1.00")
+            .version("1.02")
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/location/LocationController.java]: `1.00` -> `1.02`? What happened to `1.01`?

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/medicationdispense/MedicationDispenseController.java [hunk 2]
```diff
@@ -77,12 +77,16 @@ public MedicationDispense read(@PathVariable("publicId") String publicId) {
...
+   * The XML should remain the same, but the version of the resource needs to be incremented for
+   * SQL52.
+   */
   private CdwMedicationDispense100Root search(MultiValueMap<String, String> params) {
     Query<CdwMedicationDispense100Root> query =
         Query.forType(CdwMedicationDispense100Root.class)
             .profile(Query.Profile.DSTU2)
             .resource("MedicationDispense")
-            .version("1.00")
+            .version("1.02")
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/medicationdispense/MedicationDispenseController.java]: `1.00` -> `1.02`? What happened to `1.01`?

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitioner/PractitionerController.java [hunk 3]
```diff
@@ -78,12 +78,16 @@ public Practitioner read(@PathVariable("publicId") String publicId) {
...
+   * The XML should remain the same, but the version of the resource needs to be incremented for
+   * SQL52.
+   */
   private CdwPractitioner100Root search(MultiValueMap<String, String> params) {
     Query<CdwPractitioner100Root> query =
         Query.forType(CdwPractitioner100Root.class)
             .profile(Profile.DSTU2)
             .resource("Practitioner")
-            .version("1.00")
+            .version("1.02")
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitioner/PractitionerController.java]: `1.00` -> `1.02`? What happened to `1.01`?

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/OrganizationController.java [hunk 4]
```diff
@@ -78,12 +78,16 @@ public Organization read(@PathVariable("publicId") String publicId) {
...
+   * The XML should remain the same, but the version of the resource needs to be incremented for
+   * SQL52.
+   */
   private CdwOrganization100Root search(MultiValueMap<String, String> params) {
     Query<CdwOrganization100Root> query =
         Query.forType(CdwOrganization100Root.class)
             .profile(Profile.DSTU2)
             .resource("Organization")
-            .version("1.00")
+            .version("1.02")
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/OrganizationController.java]: `1.00` -> `1.02`? What happened to `1.01`?

**Comment by jhulbert-va** [comment]
@bschofield-va Alex and I talked and when I asked what version to make these resources, he said he was just going to move them all to `1.02`.

**Pull request merged**